services/director-v2/src/simcore_service_director_v2/modules/comp_scheduler/dask_scheduler.py

```diff
@@ -355,12 +354,22 @@ async def _task_log_change_handler(self, event: str) -> None:
         with log_catch(_logger, reraise=False):
             task_log_event = TaskLogEvent.parse_raw(event)
             _logger.debug("received task log update: %s", task_log_event)
-            *_, user_id, project_id, node_id = parse_dask_job_id(task_log_event.job_id)
             await publish_service_log(
                 self.rabbitmq_client,
-                user_id=user_id,
-                project_id=project_id,
-                node_id=node_id,
+                user_id=task_log_event.task_owner.user_id,
+                project_id=task_log_event.task_owner.project_id,
+                node_id=task_log_event.task_owner.node_id,
                 log=task_log_event.log,
                 log_level=task_log_event.log_level,
             )
+            if task_log_event.task_owner.has_parent:
+                assert task_log_event.task_owner.parent_project_id  # nosec
+                assert task_log_event.task_owner.parent_node_id  # nosec
+                await publish_service_log(
```

Comment:
Q: this is sending the logs to the logs to the frontend's logger, in case I have started the job from a dynamic service?

Reply:
yes correct